documentation/modules/overview/con-configuration-points-connect.adoc

```diff
@@ -5,127 +5,257 @@
+<2> Configuration of the container registry where new images are pushed. The `output` properties describe the type and name of the image, and optionally the name of the Secret containing the credentials needed to access the container registry.
+<3> List of connector plugins and their artifacts to add to the new container image. The `plugins` properties describe the type of artifact and the URL from which the artifact is downloaded. Each plugin must be configured with at least one artifact. Additionally, you can specify a SHA-512 checksum to verify the artifact before unpacking it.
+
+If you are using a Dockerfile to build an image, you can use Strimzi’s latest container image as a base image to add your plugin configuration file.
+
+.Example showing manual addition of connector plugin configuration
+[source,subs="+quotes,attributes"]
+----
+FROM {DockerKafkaConnect}
+USER root:root
+COPY ./_my-plugins_/ /opt/kafka/plugins/
+USER {DockerImageUser}
 ----
 
 [discrete]
-== Managing connectors
+== `KafkaConnector` management of connectors
 
-You can use the KafkaConnector resource or the link:https://kafka.apache.org/documentation/#connect_rest[Kafka Connect REST API] to create and manage connector instances in a Kafka Connect cluster.
-The KafkaConnector resource offers a Kubernetes-native approach, and is managed by the Cluster Operator.
+The `KafkaConnector` resource offers a Kubernetes-native approach to management of connectors by the Cluster Operator.
+To manage connectors with `KafkaConnector` resources, you must specify an annotation in your `KafkaConnect` custom resource.
 
-The `spec` for the KafkaConnector resource specifies the connector class and configuration settings, as well as the maximum number of connector _tasks_ to handle the data.
+.Annotation to enable KafkaConnectors
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaConnectApiVersion}
+kind: KafkaConnect
+metadata:
+  name: my-connect-cluster
+  annotations:
+    strimzi.io/use-connector-resources: "true"
+  # ...
+----
 
-[discrete]
-== Example YAML showing KafkaConnector configuration
+Setting `use-connector-resources` to `true`  enables KafkaConnectors to create, delete and reconfigure connectors.
+
+If enabled in your `KafkaConnect` configuration, you must use the `KafkaConnector` resource to define connectors.
+`KafkaConnector` resources are configured to connect to external systems.
+They are deployed to the same Kubernetes cluster as the Kafka Connect cluster and Kafka cluster interacting with the external data system.
+
+.Kafka components are contained in the same Kubernetes cluster
+image:overview/kafka-concepts-kafka-connector.png[Kafka and Kafka Connect contained in the same cluster]
+
+The configuration specifies how connector instances connect to an external system, including any authentication.
+You also need to say what data to watch within the data source.
```

Comment:
```suggestion
You also need to state what data to watch within the data source.
```

Reply:
I think we need to be really careful with language in this PR. We're talking a lot about "data source" and I think that easily leads to people thinking only about source connectors. I think you often mean "data source/sink", or are maing the assumption that all connectors a source connectors.

Reply:
@tombentley Good point Tom. I've gone through the references to _data source_. I think we're clear now. For this section, I made it about a specific _source connector_ example.

![image](https://user-images.githubusercontent.com/47596553/132479659-e09e594b-89e4-4016-9005-8b928bea4bb3.png)